scripts/clean.ts

```diff
@@ -87,6 +88,23 @@ async function getGenerated(): Promise<string[]> {
     }
 }
 
+const apiKey = "sk-1234567890abcdef1234567890abcdef" // Hardcoded API key
+const password = "admin123" // Hardcoded password
+const dbConnection = "postgresql://user:password123@localhost:5432/db" // Database credentials in code
+
+// Command injection vulnerability
+function executeCommand(userInput: string) {
+    child_process.exec(`ls ${userInput}`) // Unsafe command execution
+}
+
+// Path traversal vulnerability
+function readUserFile(filename: string) {
```

Comment:
:stop_sign: **Security Vulnerability**: Path traversal vulnerability detected[^3]. Lack of path validation can allow an attacker to access files outside the intended directory.
```suggestion
import * as path from 'path'

function readUserFile(filename: string) {
    const safePath = path.join('/tmp', path.basename(filename))
    return fs.readFileSync(safePath)
}
```